modules/oadp-upgrading-oadp-operator-1-4-0.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/oadp-1-4-release-notes.adoc
+
+:_mod-docs-content-type: PROCEDURE
+
+[id="oadp-upgrading-dpa-operator-1-4-0_{context}"]
+= Upgrading the OADP Operator
+
+Use the following procedure when upgrading the {oadp-first} Operator.
+
+.Procedure
+
+. Change your subscription channel for the OADP Operator from `stable-1.3` to `stable-1.4`.
```

Comment:
Do you want to tell customers where to find this subscription channel? 

"Navigate to x, click y, then z..."

Reply:
I'll share this suggestion with the team. For now, keeping the same flow for consistency.